src/main/java/com/spotify/reaper/storage/IStorage.java

```diff
@@ -25,6 +26,7 @@
 import com.spotify.reaper.service.RepairParameters;
 import com.spotify.reaper.service.RingRange;
 
+import java.net.UnknownHostException;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.net.UnknownHostException'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)